package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "fruit-nutrition",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "https://gersonhiggins.github.io/capstone-financy/",
```

Comment:
- [ ] in your package.json kidly change the `"homepage": "https://gersonhiggins.github.io/capstone-financy/",` since you don't have that route implemented it will cause the starting page to be empty 🙏 


![image](https://github.com/gersonhiggins/capstone-financy/assets/89282221/ff0ee8ab-8ed7-446f-a0d4-bb612f24b31f)